device-types/Fortinet/FG-2200E.yaml

```diff
@@ -0,0 +1,92 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2200E
+slug: fortinet-fg-2200e
+part_number: FG-2200E
+u_height: 2
+is_full_depth: false
+weight: 18.2
+weight_unit: kg
+airflow: front-to-rear
```

Comment:
Not sure what happened to the power, can you want to add those back as module bays?

Reply:
done